### ISSUE TITLE
No Name property is defined for the tabs controls present in network Tab panel.: A11y_Playwright OSS releases_Name Role Value

### ISSUE THREAD

**Issue by ganagdhar123** [microsoft/playwright.dev#1806]
"[Check out Accessibility Insights](https://nam06.safelinks.protection.outlook.com/?url=https://accessibilityinsights.io/&data=02%7c01%7cv-stfe%40microsoft.com%7cb67b2c4b646d4f9561a208d6f4b5c39b%7c72f988bf86f141af91ab2d7cd011db47%7c1%7c0%7c636965458850501301&sdata=mxhokIKNMb22llsjXHLgU3XZibj1Qfx37rpY4PU2sfE%3D&reserved=0)! - Identify accessibility bugs before check-in and make bug fixing faster and easier."
### GitHub Tags:
#A11yTCS;#Playwright OSS Releases;#BM_PlaywrightOSSReleases_External_Sep2025;#ChromiumEdge;#DesktopWeb;#FTP;#Win11;#A11ySev2;#A11yMAS;#WCAG4.1.2;
### Environmental Details:
App Name: Playwright OSS releases
#URL: https://playwright.dev/
OS & Browser Details:
Microsoft Windows 11 Enterprise
Version 26100.3323
Microsoft Edge
Version 131.0.2903.99 (Official build) (64-bit)
### Repro Steps:

1. Hit the URL: https://playwright.microsoft.com/workspaces/eastus_cdbdcaba-d6e1-47a0-80b2-e586bb4a7247 and sign in with workspace account.
2. "Home" page will appear. TAB to any "Test run" and hit ENTER key.
3. TAB to any test case and hit ENTER key. TAB to "View trace viewer" button and hit ENTER key.
4. TAB to "Network" tab and hit ENTER key.
5. Tab till "All", "Fetch" etc. Tabs under Network tab panel.
6. Observe that name property is defined for controls present in Network Tab panel or not.

### Actual:
No name property is defined for ("All", "fetch", "HTML", "JS", "CSS", and "Images") controls present in Network Tab panel.
### Expected:
Name property should be provided for the ("All", "fetch", "HTML", "JS", "CSS", and "Images") controls present in Network Tab panel.
### User Impact:
Screen reader users will not be able to access the controls if the name property in not defined for the Tab controls.
### Attachment:

<img width="1279" height="668" alt="Image" src="https://github.com/user-attachments/assets/b987f646-a3ec-4487-b0c1-1d01f60be863" />
